android/TritonClient/tritonlib/src/main/java/com/mobiledgex/tritonlib/ImageSenderTriton.java

```diff
@@ -261,7 +262,8 @@ private void handleResponse(String response, long latency) {
 
             } else if (mModelName.equals("ensemble_dali_inception")) {
                 Log.i(TAG, "ensemble_dali_inception response="+response);
-                String inference = response.substring(239);
+                // Add 4 to ignore binary data before the actual inference data.
+                String inference = response.substring(INCEPTION_INFERENCE_HEADER_CONTENT_LENGTH+4);
```

Comment:
👍 
Do you have  the REST API References link for Dali_Inception and Dali_YOLO4? 
The response and the request structure and options basically.

Reply:
Not yet. Created https://mobiledgex.atlassian.net/browse/EDGECLOUD-5228 to track this.